### DISCUSSION TITLE
How to change the neovim version?

### DISCUSSION THREAD

**Discussion by kaushiknfs** [neovide/neovide#986] [Q&A] [answered]
Hi,
I've neovim 0.5.0 in my system, neovim inside shows its version 0.4.3 , how do i change that?

Thanks.

**Comment by kaushiknfs** [comment, marked as answer]
It got fixed when i installed neovim also with scoop.

**Reply by Kethku** [reply]
Neovide just selects the first nvim.exe on your path. You likely have multiple neovim installs
